packages/analytics/dashboard-renderer/sandbox/sandbox-query-provider.ts

```diff
@@ -1,7 +1,7 @@
 import type { Plugin } from 'vue'
 import { nonTsExploreResponse, routeExploreResponse } from './mock-data'
 import { INJECT_QUERY_PROVIDER } from '../src'
-import { generateSingleMetricTimeSeriesData, type AnalyticsBridge, type AnalyticsConfigV2, type ExploreQuery, type ExploreResultV4 } from '@kong-ui-public/analytics-utilities'
+import { generateSingleMetricTimeSeriesData, type AnalyticsBridge, type AnalyticsConfigV2, type DatasourceAwareQuery, type ExploreQuery, type ExploreResultV4 } from '@kong-ui-public/analytics-utilities'
```

Comment:
This is not mission critical since it's the sandbox, but can you remove `ExploreQuery` from the import since it's been replaced?